includes/wp-admin/class-settings-fields.php

```diff
@@ -466,4 +482,27 @@ public static function render_authorized_fetch_field() {
 		</p>
 		<?php
 	}
+
+	/**
+	 * Render relays field.
+	 */
+	public static function render_relays_field() {
+		$value = get_option( 'activitypub_relays', array() );
+		?>
+		<p>
+			<?php echo wp_kses( __( 'A <strong>Fediverse-Relay</strong> distributes content across instances, expanding reach, engagement, and discoverability, especially for smaller instances.', 'activitypub' ), 'default' ); ?>
+		</p>
```

Comment:
How would you feel about moving this below the textarea? In previous PRs I noticed there's no margins between this text and the input box.

Reply:
I thought that this might be a relevant info you should read before filling out stuff!?!

Reply:
I do hope users read descriptions regardless of where they're placed